changelogs/fragments/597-openssh_keypair-test-key-creation.yml

```diff
@@ -0,0 +1,2 @@
+trivial:
+  - "openssh_keypair - test creating new keys with all values of ``regenerate``"
```

Comment:
We generally do not add changelog entries for test changes; please remove this file.

Reply:
Done in the latest push